index.html

```diff
@@ -1225,6 +1225,31 @@ <h3>Co-signed</h3>
               <td>Company</td>
               <td>Development; DevOps; open-source community efforts</td>
             </tr>
+            <tr>
+              <td><a href="https://sredevops.org">SREDevOps.org</a></td>
+              <td>Company</td>
+              <td>SRE, DevOps and Platform Engineering Community</td>
+            </tr>
+            <tr>
+              <td><a href="https://kineo.ai/">Kineo.ai</a></td>
+              <td>Company</td>
+              <td>Open-source community efforts</td>
+            </tr>
+            <tr>
+              <td><a href="https://nypl.org">The New York Public Library</a></td>
+              <td>Company</td>
+              <td>Development; open-source community efforts</td>
+            </tr>
+            <tr>
+              <td><a href="https://strike.sh">Mantle</a></td>
```

Comment:
@flavius-dinu This might be a accidental merge/copy from [my previous PR](https://github.com/opentffoundation/manifesto/pull/1004). Happy to open a PR if helpful!